libfwupdplugin/fu-common.h

```diff
@@ -67,6 +67,7 @@ typedef guint FuEndianType;
  * @FU_PATH_KIND_SYSFSDIR_SECURITY:	The sysfs security location (IE /sys/kernel/security)
  * @FU_PATH_KIND_ACPI_TABLES:		The location of the ACPI tables
  * @FU_PATH_KIND_LOCKDIR:		The lock directory (IE /run/lock)
+ * @FU_PATH_KIND_FIRMWARE_SEARCH:	The path to configure the kernel policy for runtime loading other than /lib/firmware (IE /sys/module/firmware_class/parameters/path)
```

Comment:
This seems super Linux-specific, no?

Reply:
Yes it is. Let's see if it's useful for modem manager plugin for now first before deciding what to do with it.